### PR TITLE
Unresolved Call Summarization Example

### PR DESCRIPTION
**Pull request by nivcertora**

### CODE REVIEW

File: CVLByExample/UnresolvedCallSummarization/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Unresolved Call Summarization Example
+
+## Overview
+This repository contains a smart contract named `TrusterLenderPool`, featuring a `flashLoan` function. However, there exists a significant vulnerability within this function. The vulnerability allows the pool contract to act as the `msg.sender` for any call, presenting a critical security flaw. This example showcases the utilization of a new feature in CVL called Unresolved Call Summarization. This feature accentuates how the vulnerability can be exploited to grant the attacker an infinite allowance, subsequently leading to the potential draining of all funds within the contract.
```
Comment (nd-certora) [inline CVLByExample/UnresolvedCallSummarization/README.md:4]: add a note that this is based on damnVulnerability and give a link

File: CVLByExample/UnresolvedCallSummarization/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- If the signature call matches the `approve` function, it is categorized as `token.approve(address, uint256)`.
+- If the signature call matches any of the functions within `currentContract`, their implementations are utilized.
+- Otherwise, it triggers a `NONDET` (nondeterministic) summary.
+
+## Executions
+
+1. **Certora Run Command**
+    - Command:
+        ```bash
+        certoraRun truster.conf --server production --prover_version master
```
Comment (nd-certora) [inline CVLByExample/UnresolvedCallSummarization/README.md:24]: no need --server production --prover_version master we will post when it is in production

File: CVLByExample/UnresolvedCallSummarization/README.md (hunk 1 above)
Comment (nd-certora) [reply CVLByExample/UnresolvedCallSummarization/README.md:4]: also can we estimate what will be the link in the documentation, this is the PR:
https://github.com/Certora/Documentation/pull/236

**Review by nd-certora — APPROVED** [review]
small comments

File: CVLByExample/UnresolvedCallSummarization/README.md (hunk 1 above)
Comment (nivcertora) [reply CVLByExample/UnresolvedCallSummarization/README.md:4]: Done

File: CVLByExample/UnresolvedCallSummarization/README.md (hunk 2 above)
Comment (nivcertora) [reply CVLByExample/UnresolvedCallSummarization/README.md:24]: Done

**Pull request merged**